Source/WebCore/rendering/RenderLayer.h

```diff
@@ -1032,6 +1032,7 @@ class RenderLayer final : public CanMakeSingleThreadWeakPtr<RenderLayer>, public
         Seen3DTransformedLayer              = 1 << 7,
         SeenCompositedScrollingLayer        = 1 << 8,
         SubtreeNeedsUpdate                  = 1 << 9,
+        EnvironmentChanged                  = 1 << 10,
```

Comment:
I think this needs some comment. it's not super obvious what "environment" here means.